libpod/container_internal.go

```diff
@@ -131,14 +133,92 @@ func (c *Container) AttachSocketPath() string {
 	return filepath.Join(c.runtime.ociRuntime.socketsDir, c.ID(), "attach")
 }
 
+// exitFilePath gets the path to the container's exit file
+func (c *Container) exitFilePath() string {
+	return filepath.Join(c.runtime.ociRuntime.exitsDir, c.ID())
+}
+
+// create a bundle path and associated files for an exec session
+func (c *Container) createExecBundle(sessionID string) (err error) {
+	bundlePath := c.execBundlePath(sessionID)
+	if createErr := os.MkdirAll(bundlePath, execDirPermission); createErr != nil {
+		return createErr
+	}
+	defer func() {
+		if err != nil {
+			if err2 := os.Remove(bundlePath); err != nil {
```

Comment:
Doesn't this have to be a `RemoveAll()`?

Reply:
I am pretty sure `RemoveAll` is for nested directories, of which I don't think there are any here. I can change it though just in case there becomes any